4-fleetscope/envs/development/terraform.tfvars

```diff
@@ -0,0 +1 @@
+../../terraform.tfvars
```

Comment:
From the lint error, I think these might be files rather than symlinks?  I'm not sure if there is a way to tell from the PR UI?